angular/package.json

```diff
@@ -25,6 +25,7 @@
     "@angular/cli": "^20.0.0-next.7",
     "@angular/compiler-cli": "^20.0.0-next.8",
     "@types/jasmine": "~5.1.0",
+    "@types/node": "^20",
```

Comment:
We also have v18 declared somewhere so now we have both, and if you look at the lockfile we now have multiple odd combos of peer deps (like `@angular/cli` with `@types/node` 18 and then again with 20).

I guess it doesn't really matter, but it would be nice to cleanup the lockfile. Either change this to 18, change the 18 to 20, or do nothing and align versions another time...

Reply:
I think this repo should represent a real-world scenario, and it's common that teams don't align. So I don't think we should spend time on anything cleanup-shaped unless most of our customers are doing that too.

Reply:
But seeing a million combos of peer deps in a lockfile makes me 😡 

(but you are correct and no action is needed here)

Reply:
To solve the problem you point out, users should use one of the existing tools that aligns package.json files. Want to write a blog post on it? That's sometimes my approach to dealing with that frustration 🫣